napari/layers/shapes/_tests/test_shapes_mouse_bindings.py

```diff
@@ -17,20 +18,27 @@
 )
 
 
-@pytest.fixture
-def Event():
-    """Create a subclass for simulating vispy mouse events.
+@dataclass
+class Event:
```

Comment:
```suggestion
class MouseEvent:
```
To be more clear that it is mouse event, not general event